# Prediction of disassembly units in protein shells

## Introduction
The combination of elastic network models (ENM), normal mode analysis (NMA), and vibrational cluster decomposition allows the identification of quasi-rigid mechanical units in protein shells and other macromolecular complexes ([Polles *et al.*, 2013](https://doi.org/10.1371/journal.pcbi.1003331);[Ponzoni *et al.*, 2015](https://doi.org/10.1016/j.str.2015.05.022)). These prior studies interpreted these mechanical units as the potential assembly subunits. But, viral capsids are often subject to maturation processes that alter the capsid mechanically, including protein cleavage, protein allostery, and the addition of reinforcement proteins ([Bruinsma, Wuite, and Roos](https://doi.org/10.1038/s42254-020-00267-1);[Perlmutter and Hagan](https://doi.org/10.1146/annurev-physchem-040214-121637)). Therefore, the quasi-rigid units identified by pyCapsid and analogous approaches can be interpreted as the mechanical units that are more likely to represent the capsid regions involved in the initial steps of the disassembly of a capsid (or a macromolecular complex). To test the use of pyCapsid as an approach to identify the potential disassembly units, we analyzed seven viral capsids that had been reported to (completely or partially) disassemble experimentally in controlled conditions *in vitro*.

## Methods

**Viral capsid selection.** The capsids selected for analysis satisfied the following premises. First, there was a molecular reconstructed model in the protein data bank (PDB). Second, it had been studied as a canonical model system for assembly/disassembly (like hepatitis B virus), or experiments showing partial or complete disassembly could be found. The screening was not exhaustive. Instead, we compromised on having representatives of capsids formed by major capsid proteins adopting different folds and storing different types of genomes. The focus of this initial study was representatives of small and medium capsid sizes. A list of seven capsid representatives is described in more detail below. The major capsids of three representatives adopt the jelly-roll fold, two the HK97-fold, one the alpha-helical fold, and one the beta-unique fold. Four of the viruses store their genome in the form of RNA, while the other three store their genome in the form of DNA. Based on the role of disassembly in the life cycle and experimental observations, the capsids were organized into two groups: Group 1 included capsids that disassemble as part of the virus life cycle; Group 2 included capsids that do not disassemble during the virus life cycle. In the second group, the capsids were divided into two subgroups depending on whether the capsids that were observed to completely (Group 2.1) or partially (Group 2.2) disassemble *in vitro*. The list below includes the viral capsid name, PDB ID, virus taxonomy (realm, order, and family), virus-host, capsid description, protein fold, disassembly in the life cycle, the *in vitro* disassembly method, disassembly units observed *in vitro*, references, and the disassembly group assigned in this study:

+ **Seneca Valley virus (SSV) capsid:** PDBid [3cji](https://doi.org/10.2210/pdb3CJI/pdb). Taxonomy: *Riboviria*, *Picornavirales*, *Picornaviridae*.
The Seneca Valley virus (SSV) infects pigs (porcine virus) and has been repurposed as an oncolytic agent because it can infect human cancer cells ([Reddy *et al.*, 2007](https://doi.org/10.1093/jnci/djm198)). The structure analyzed corresponded to the capsid of the SVV-001 virion, which has cytotoxicity towards tumor cells ([Venkatamaran *et al.* 2008](https://doi.org/10.1016/j.str.2008.07.013)). The capsid was reconstructed at 2.30 Å resolution by X-ray crystallography. The capsid contains 240 capsid proteins distributed as 60 copies of four different proteins (VP1, VP2, VP3, and VP4). VP1, VP2, and VP3 correspond to the major capsid proteins. They are similar and adopt the single-jelly roll (with eight beta-strands arranged in a barrel-like structure). ([Venkatamaran *et al.* 2008](https://doi.org/10.1016/j.str.2008.07.013)). In the life cycle, it has been observed that a minor capsid protein, VP4, is removed, providing a pore where the genome is translocated, so the capsid does not necessarily fully disassemble as part of the life cycle. The *in vitro* disassembly experiments observed that acidic conditions promoted the disassembly: the virion disassembled at pH ≤ 5.5 and procapsids at pH ≤ 6.0 in phosphate-buffered saline (PBS) ([Strauss *et al.* 2018](https://doi.org/10.1128/jvi.01927-17)). The experiments identified 5-fold symmetric clusters, each containing 15 major capsid proteins as the dissociation units. The authors named the disassembly units pentamers because the major capsid proteins exist as three variants (VP1, VP2, and VP3), and there are five of each in the so-called pentamer ([Strauss *et al.* 2018](https://doi.org/10.1128/jvi.01927-17)). However, here, we will refer to them as pentons because pentamer is typically used for clusters with five proteins. Disassembly Group 2.1. 

+ **Triatoma virus (TrV) capsid:** PDBif [3nap](https://doi.org/10.2210/pdb3NAP/pdb). Viral taxonmoy: *Riboviria*, *Picornavirales*, *Dicistroviridae*. This virus infects (triatomine) insects (kissing bugs), which are vectors of human trypanosomiasis (Chagas disease). Thus, TrV is a potential biological agent of interest to reduce Chagas disease. The capsid structure analyzed was resolved at 2.5 Å resolution by X-ray crystallography ([Squires *et al.* 2013](https://doi.org/10.1107%2FS0907444913004617)). The capsid contains 240 proteins that include 60 copies of four different proteins: Three variants of the major capsid protein (VP1, VP2, and VP3) and a minor protein (VP4). The major capsid proteins adopt the jelly roll fold ([Squires *et al.* 2013](https://doi.org/10.1107%2FS0907444913004617)). The virus does not rely on capsid disassembly to infect the host. But capsid disassembly was observed *in vitro* by heating the TrV virions to 55°C for 30 minutes and then storing the samples for a week at 4°C ([Agirre *et al.* 2013](https://doi.org/10.1099/vir.0.048553-0)). Cryo-electron dissociation units were visualized by cryo-EM, revealing 5-fold symmetrical pentons (15 MCPs). The pentons were observed to form penton-penton lip-shaped particles (LPs). Disassembly Group 2.1.

+ **Hepatitis B virus (HBV) capsid:** PDBid [2g33](https://doi.org/10.2210/pdb2G33/pdb). Taxonomy: *Riboviria*, *Blubervirales*, *Hepadnaviridae*. This human virus primarily infects liver cells (hepatocytes) and is a cause of liver cancer. The capsid analyzed was resolved at 3.96 Å resolution by X-ray crystallography ([Bourne, Finn, and Zlotnick, 2006](https://doi.org/10.1128/JVI.00933-06)). The capsids are formed by 420 major capsid proteins, each adopting the alpha-helical fold ([Wynne, Crowther, and Leslie, 1999](https://doi.org/10.1016/S1097-2765(01)80009-5)). In the HBV life cycle, it is assumed that the capsid disassembles, releasing the viral genome into the nuclear basket ([Schädler and Hildt, 2009](https://doi.org/10.3390%2Fv1020185)). Capsids have been disassembled *in vitro* using urea, observing dimers of the major capsid protein with mass spectrometry ([Starr *et al.*, 2022](https://doi.org/10.1021/acs.biochem.1c00810)). Disassembly Group 1. 

+ **Cowpea chlorotic mottle virus (CCMV) capsid:** PDBid [1za7](https://doi.org/10.2210/pdb1ZA7/pdb). Viral taxonomy: *Riboviria*, *Martellivirales*, *Bromoviridae*. Virus infecting the cowpea plant (or black-eyed pea). The capsid analyzed was a salt-stable mutant of the CCMV capsid that is not susceptible to high-salt and was revolved at 2.70 Å resolution by X-ray crystallography ([Speir *et al.*, 2006](https://doi.org/10.1128/JVI.80.7.3582-3591.2006)). The native capsid of CCMV, PDB [1cwp](https://doi.org/10.2210/pdb1CWP/pdb) published by [Speir *et al.*, 1995](https://doi.org/10.1016/s0969-2126(01)00135-6), was also analyzed, but the empirical B-factors were uniform and didn't allow to calibrate the scale of the pyCapsid predictions, so the results were not used for the analysis. The salt-stable capsid analyzed is formed by 180 copies of the same major capsid protein, each adopting the single jelly roll fold. In the CCMV life cycle, the capsid is assumed to dissociate to release the genome, but it is unclear if instead the capsid goes through a swollen state without fully disaggregating ([Wilts, Schaap, and Schmidt, 2015](https://doi.org/10.1016%2Fj.bpj.2015.04.019)). The wild-type capsids of CCMV have been disassembled at high salt conditions and pH above 7, observing dimers as the disassembly units ([Bancroft, Hills, Markham, 1967](https://doi.org/10.1016/0042-6822(67)90180-8) or monomers when including DDT (Dithiothreitol) and (phenylmethylsulfonylfluoride) to prevent disulfide bonds and inhibit proteases, respectively ([Lavelle, Michel, and Gingery, 2007](https://doi.org/10.1016/j.jviromet.2007.07.020)). Disassembly Group 1 / 2.1 (unclear).

+ **Bacteriophage MS2 capsid:** PDBid [2ms2](https://doi.org/10.2210/pdb2MS2/pdb) ([REF](https://doi.org/10.1006/jmbi.1993.1616)). Taxonomy: *Riboviria*, *Norzivirales*, *Fiersviridae*. Virus (bacteriophage) infecting the bacterium *Escherichia coli* and other members of the same family. It is also called *Ermesvirus zinderi*. The capsid analyzed was reconstructed at 2.80 Å resolution by X-ray crystallography ([Golmohammadi *et al.*, 1993](https://doi.org/10.1006/jmbi.1993.1616)). In MS2's life cycle, the genome is released in the cell via the interaction of the mature protein with the F-pilus, and the capsid remains outside the cell ([Valentine and Wedel, 1965](https://doi.org/10.1016/0006-291x(65)90094-x);[Harb *et al.*, 2020](https://doi.org/10.1073/pnas.2011901117)). The disassembly of MS2 capsids was studied at 5 mM sodium phosphate buffer at pH 70.0 and 4.5 using nuclear magnetic resonance (NMR), observing dimers as the dissociation units ([Anobom *et al.*, 2003](https://doi.org/10.1016%2FS0006-3495(03)75117-0)). Disassembly Group 2.1.

+ **Salmonella virus P22 mature capsid:** PDBid [5uu5](https://doi.org/10.2210/pdb5UU5/pdb). Taxonomy: *Duplodnaviria*, *Caudovirales*, *Podoviridae*. Virus (bacteriophage) infecting the bacterium *Salmonella typhimurium*, a major agent of foodborne illness. The virus is commonly referred to as phage P22. The structure analyzed corresponded to the mature capsid from the virion, which had been reconstructed at 3.30 Å resolution by cryo-electron microscopy ([Hryc *et al.*, 2017](https://doi.org/10.1073/pnas.1621152114)). The reconstructed capsid contains 420 major capsid proteins adopting the HK97-fold, although the actual virion replaces five of those proteins located on a 5-fold vertex by the tail apparatus. In phage P22's life cycle, the genome is translocated into the cell by the phage tail and an internal protein complex that is externalized upon infection, while the capsid remains stable outside the bacteria ([Juin *et al*, 2015](https://doi.org/10.1016/j.virol.2015.07.006),[Lander *et al.*, 2009](https://doi.org/10.1016%2Fj.str.2009.04.006)). The partial disassembly of mature-like capsids of P22 was studied *in vitro* using the expanded shell capsid heated at 75°C for 20 min, causing the release of pentamers from fivefold axes, producing wiffleball capsids ([Kant *et al.*, 2018](https://doi.org/10.1016/j.bbagen.2018.03.006)). Disassembly Group 2.1.

+ ***Escherichia* virus Hong Kong (HK97) mature capsid:** PDBid [2ft1](https://doi.org/10.2210/pdb2FT1/pdb). Taxonomy: *Duplodnaviria*, *Caudovirales*, *Siphoviridae*. *Escherichia* virus HK97 is a bacteriophage that infects the bacterium *Escherichia coli* and related bacteria. The HK acronym in the name refers to the isolation place, Hong Kong. The structure analyzed corresponded to the mature capsid (Head II state) reconstructed at 3.90 Å resolution by X-ray crystallography ([Gan *et al.*, 2006](https://doi.org/10.1016/j.str.2006.09.006)). The reconstructed capsid contains 420 major capsid proteins adopting the HK97-fold, although the actual virion replaces five of those proteins located on a 5-fold vertex by the tail apparatus. In the phage HK97's life cycle, the genome is translocated into the cell by the phage tail, while the capsid remains stable outside the bacteria as in other tailed phages ([Hendrix and Johnson, 2012](https://doi.org/10.1007/978-1-4614-0980-9_15)). The disassembly of the mature capsid was observed at high temperatures, around 95°C ([Duda *et al.*, 2009](https://doi.org/10.1016/j.jmb.2009.06.035)). The measurements obtained by scanning calorimetry indicated that the denaturation of the capsid and its subunits are coupled processes: "After a certain threshold temperature is passed, any given particle sustains, stochastically, an initial local lesion. Thereafter, damage spreads out from that site where subunits are no longer fully surrounded by stabilizing inter-subunit interactions" ([Duda *et al.*, 2009](https://doi.org/10.1016/j.jmb.2009.06.035)). Disassembly Group 2.1.

**Identification of disassembly units.** The pyCapsid package was applied to analyze the viral capsids and identify quasi-rigid regions in the capsids, which were interpreted as potential disassembly units. The unified elastic network model (U-ENM), also known as the generalized anisotropic network model (G-ANM), was applied to all capsids and calibrated with the experimental B-factors. The default number of modes (200) was increased systematically to guarantee predicted B-factors mimicking the overall empirical B-factors measured across amino-acid residues and reaching a correlation coefficient above 0.5. Within pyCapsid, the default clustering method (spectral clustering) was applied to screen clusters ranging from ten to three times the number of capsid proteins. The standard quasi-rigid cluster coefficient score defined in ([Ponzoni *et al.*, 2015](https://doi.org/10.1016/j.str.2015.05.022)) was used to identify quasi-rigid mechanical units (peaks) as a function of the number of clusters. If more than one peak (local maximum) was observed, the optimal clustering configurations were ranked by score value (descending order). The quasi-rigid domains were visualized using the pyCapsid module to generate results to be displayed in ChimeraX, and the standard pyCapsid output report was generated. The identified quasi-rigid mechanical units were compared with the protein complexes observed in disassembly experiments. For those capsids where the disassembly was partial, the predictions from pyCapsid were filtered as a function of the intensity of B-factors (quasi-rigid units associated with larger B-factors were interpreted as more prone to disassembly first).

## Results and Discussion

+ **Seneca Valley virus (SVV) capsid:** PDBid [3cji](https://doi.org/10.2210/pdb3CJI/pdb). In the pyCapsid analysis, the number of modes was increased to 800, reaching a correlation coefficient of CC = 0.64 between empirical and predicted B-factors. The predicted B-factor profile displayed larger fluctuations in a few small regions of about ~15 amino acids, probably due to the presence of RNA in the capsids, which was not included in the PDB or in pyCapsid's model. The clustering analysis revealed a prominent peak associated with the presence of 12 clusters of a unique type. The visual analysis identified these clusters as pentons containing 15 MCPs (five copies of each of the three variants, VP1, VP2, and VP3). These pentons coincide with the dissociation units observed in the disassembly experiments of the SVV capsid [Strauss *et al.* 2018](https://doi.org/10.1128/jvi.01927-17). To access the figures associated with the results summarized above, we refer the reader to the online [SSV's pyCapsid report](https://luquelab.github.io/pyCapsid/gallery/3cji_pyCapsid_report/pyCapsid_report.html).

+ **Triatoma virus (TrV) capsid:** PDBid [3nap](https://doi.org/10.2210/pdb3NAP/pdb). In the pyCapsid analysis, the number of modes was increased to 1200, reaching a correlation coefficient of CC = 0.42 between empirical and predicted B-factors. The predicted B-factor recovered the overall pattern across residues but was short in capturing relatively large fluctuations in two regions of about 20 residues around regions 0 and 250. The maximum cluster quality score was observed as a sharp peak at 12 clusters of a unique type. The visual analysis identified these clusters as pentons containing 15 MCPs (five copies of each of the three variants, VP1, VP2, and VP3). These pentons coincided with the dissociation units observed in the disassembly experiments of the capsid [Agirre *et al.* 2013](https://doi.org/10.1099/vir.0.048553-0)). To access the figures associated with the results summarized above, we refer the reader to the online [TrV's pyCapsid report](https://luquelab.github.io/pyCapsid/gallery/3nap_pyCapsid_report/pyCapsid_report.html).

+ **Hepatitis B virus (HBV) capsid:** PDBid [2g33](https://doi.org/10.2210/pdb2G33/pdb). In the pyCapsid analysis, the number of modes was increased to 800, reaching a correlation coefficient of CC = 0.57 between empirical and predicted B-factors. The predicted B-factor profile recovered the average pattern across residues but did not follow the frequent fluctuations throughout the average of the B-factor profile. The maximum cluster quality score was observed as a prominent peak at 120 clusters of a unique type. The visual analysis identified these clusters as dimers of the MCPs. The predicted dimers were consistent with the dissociated dimers observed in the capsid's disassembly experiments ([Starr *et al.*, 2022](https://doi.org/10.1021/acs.biochem.1c00810)). To access the figures associated with the results summarized above, we refer the reader to the online [HBV pyCapsid report](https://luquelab.github.io/pyCapsid/gallery/2g33_pyCapsid_report/pyCapsid_report.html).

+ **Cowpea chlorotic mottle virus (CCMV) capsid:** PDBID [1za7](https://doi.org/10.2210/pdb1ZA7/pdb). In the pyCapsid analysis, the number of modes was increased to 5738, reaching a correlation coefficient of CC = 0.56 between experimental and predicted B-factors. The predicted B-factor profile recovered the overall pattern of the empirical signal across residues, except around the 160-170 residue region and 320-240, where the fluctuation of the model was significantly lower. The maximum cluster quality score was observed as a pronounced but shallow peak at 180 clusters with three different types of clusters. The visual analysis identified these clusters as monomers of the MCPs. Another pronounced and sharp peak was observed at a lower quality score for 32 clusters with two types of unique clusters. The visual inspection identified this clustering as groups of pentamers (5 MCPs) and hexamers (6 MCPs). The prediction of monomers as the dissociation units associated with the maximum quality score was consistent with the most recent disassembly experiments ([Lavelle, Michel, and Gingery, 2007](https://doi.org/10.1016/j.jviromet.2007.07.020)). The predicted clustering with hexamers and pentamers predicted by the secondary quality score peak is consistent with the organization of proteins in the swollen state of the CCMV capsid ([Speir *et al.*, 1995](https://doi.org/10.1016/s0969-2126(01)00135-6)). To access the figures associated with the results summarized above, we refer the reader to the online [CCMV pyCapsid report](https://luquelab.github.io/pyCapsid/gallery/1za7_pyCapsid_report/pyCapsid_report.html).

+ **Bacteriophage MS2 capsid:** PDBid [2ms2](https://doi.org/10.2210/pdb2MS2/pdb).In the pyCapsid analysis, the number of modes was increased to 800, yielding a correlation coefficient of CC = 0.66 between empirical and predicted B-factors. The predicted B-factor profile recovered the average profile of the empirical B-factor signal across residues, but the experimental values had significantly higher peaks and lower valleys. The maximum cluster quality score was observed as a pronounced peak at 90 clusters with one unique type of cluster. The visual analysis identified these clusters as dimers of the MCPs. A secondary peak was observed at 12 clusters with one unique cluster. The visual analysis identified clusters containing 10 MCPs (10-mers) organized as star-shaped pentons of dimers (CONFIRM). The prediction of dimers as the dissociation units associated with the maximum quality score was consistent with the dissociation units observed in spontaneous capsid disassembly experiments ([Anobom *et al.*, 2003](https://doi.org/10.1016%2FS0006-3495(03)75117-0)) ([Lavelle, Michel, and Gingery, 2007](https://doi.org/10.1016/j.jviromet.2007.07.020)). To access the figures associated with the results summarized above, we refer the reader to the online [MS2 pyCapsid report](https://luquelab.github.io/pyCapsid/gallery/2ms2_pyCapsid_report/pyCapsid_report.html).

+ **Salmonella virus P22 mature capsid:** PDBid [5uu5](https://doi.org/10.2210/pdb5UU5/pdb). In the pyCapsid analysis, the number of modes was increased to 8000. The optimal number of modes was 3795, yielding a correlation coefficient of CC = 0.60 between empirical and predicted B-factors. The predicted B-factor profile recovered the average profile of the empirical B-factor signal across residues, but the experimental values were noisier, although the predicted values displayed a larger fluctuation around residues 1500, and several residues separated about 400 residues from each other. There were two competing maxima in the cluster quality score profile. One corresponded to 12 clusters of a unique type. The visual inspection confirmed that these were associated with pentons made of 35 proteins each. The second maximum corresponded to 72 clusters of two different types. The visual inspection confirmed the clustering of proteins in 60 hexamers (six major capsid proteins each) and 12 pentamers (five major capsid proteins each). The existence of these two competing maxima was interpreted as a mechanical indication of the capsid reinforcement mechanism ([Parent *et al.*, 2010](https://doi.org/10.1016%2Fj.str.2009.12.014)), which is why the (partial) disassembly of the mature capsid is only observed at high-heat conditions ([Kant *et al.*, 2018](https://doi.org/10.1016/j.bbagen.2018.03.006)). The peak associated with the hexamer-pentamer clustering was compatible with the observation of pentamers being released from the mature capsid forming wiffle balls in the partial disassembly experiments. To access the figures associated with the results summarized above, we refer the reader to the online [P22 pyCapsid report](https://luquelab.github.io/pyCapsid/gallery/5uu5_pyCapsid_report/pyCapsid_report.html).

+ ***Escherichia* virus Hong Kong (HK97) mature capsid:** PDBid [2ft1](https://doi.org/10.2210/pdb2FT1/pdb). In the pyCapsid analysis, the number of modes explored was increased to 10,000, yielding a correlation coefficient of CC = 0.55 between empirical and predicted B-factors. The predicted B-factor profile recovered the average profile of the empirical B-factor signal across residues, although the experimental values were noisier. There was one dominant maximum in the cluster quality score profile that corresponded to 484 clusters with seven unique clusters. Among the 484 clusters, 420 corresponded to clusters capturing most of the core residues of the major capsid proteins. The remaining 64 clusters corresponded to small extended domains of less than ten residues. The identification of the major capsid proteins as quasi-units with some minor clusters was consistent with the observation of the capsid denaturation into denatured individual proteins at high temperatures ([Duda *et al.*, 2009](https://doi.org/10.1016/j.jmb.2009.06.035)). To access the figures associated with the results summarized above, we refer the reader to the online [HK97 pyCapsid report](https://luquelab.github.io/pyCapsid/gallery/2ft1_pyCapsid_report/pyCapsid_report.html). DONE.

## Conclusions
pyCapsid predicted the dissociation units of the seven capsids analyzed correctly. However, in several cases, the disassembly conditions were very different from the conditions where the capsid structure was reconstructed. A future step would be to modify the elastic network in pyCapsid as a function of physical conditions to be more accurate in predicting disassembly units as a function of pH, salt, and temperature.
